src/test/java/com/wikia/webdriver/testcases/articlecrudtests/ArticleFeaturesCRUDTestsUser.java

```diff
@@ -222,8 +222,7 @@ public void modifyVideo() {
   @Execute(asUser = User.USER, onWikia = "mobileregressiontesting")
   public void deleteVideo() {
     new ArticleContent()
-        .push(
-            "[[File:Wikia University - How to Contact Wikia|thumb|right|335 px]]QAWebdriverCaption");
+        .push("[[File:Top 20 Shots the Tennis World will NEVER Forget|thumb|right|335 px]]");
```

Comment:
this change is already in scope of https://github.com/Wikia/selenium-tests/pull/2203 - either remove it from this changeset, or close the other PR